ceph.spec.in

```diff
@@ -79,6 +79,10 @@ BuildRequires:	python-requests
 %if ( 0%{?rhel} > 0 && 0%{?rhel} < 7 ) || ( 0%{?centos} > 0 && 0%{?centos} < 7 )
 BuildRequires:	python-sphinx10
 %endif
+%if 0%{?fedora} || 0%{defined suse_version}
```

Comment:
else statements have funny side effects. I would rather avoid them completely. 

Testing the rpm on debian for example is less problematic if its not assumed that all platforms have builddeps that dont match a platform.

SO if its Ok with you I would rather not use an else statement.